src/integrators/wrapper_integrator.hpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2019-2020, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#ifndef WRAPPER_INTEGRATOR_HPP
+#define WRAPPER_INTEGRATOR_HPP
+
+#include "mfem.hpp"
+
+/// A class to convert linearform integrators into a nonlinear residual-based one
+class LinearNonlinearFormIntegrator : public mfem::NonlinearFormIntegrator
```

Comment:
I'm afraid this name might be a bit confusing, but I don't really have a better answer. Maybe `LinearToNonlinearFormIntegrator`?

Reply:
Sure. I can change that. I'll do the same with the other ones.